app/views/govuk_component/title.raw.html.erb

```diff
@@ -2,9 +2,9 @@
   average_title_length ||= :average
   context ||= false
 %>
-<div class="govuk-title length-<%= average_title_length %>">
+<div class="pub-c-title pub-c-title--<%= average_title_length %>">
```

Comment:
I think this means if the long title length is not active we'll have a class of "pub-c-title pub-c-title--" which is not a big deal but seems like it could be avoided?